src/main/java/com/xiaomi/infra/pegasus/client/PegasusTable.java

```diff
@@ -1173,16 +1178,13 @@ public int batchMultiGet2(
   public MultiGetSortKeysResult multiGetSortKeys(
       byte[] hashKey, int maxFetchCount, int maxFetchSize, int timeout) throws PException {
     if (timeout <= 0) timeout = defaultTimeout;
-    MultiGetSortKeysResult sortKeysResult = new MultiGetSortKeysResult();
-    sortKeysResult.keys = new ArrayList<>();
-    ScanOptions options = new ScanOptions();
-    options.noValue = true;
-    ScanRangeResult result = scanRange(hashKey, null, null, options, maxFetchCount, timeout);
-    for (Pair<Pair<byte[], byte[]>, byte[]> pair : result.results) {
-      sortKeysResult.keys.add(pair.getLeft().getValue());
-    }
-    sortKeysResult.allFetched = result.allFetched;
-    return sortKeysResult;
+    ScanOptions scanOptions = new ScanOptions();
```

Comment:
should we reserver the old interface and add a new interface for these code?

Reply:
Actually, if like `multiGet`, origin `multiGetSortKeys` should also be retained and only marked `Deprecated`. but the API has been refactored by `scan` in previous PR and some user has use it, here I just keep it.  @neverchanje @hycdong can give some suggestion

Reply:
I think it is okay to update code in function multiGetSortKeys, because it is already implemented by scan.